docs/changelog.md

```diff
@@ -5,8 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Prepare and run action
 
-## [0.1.0]
+## [v0.1.0]
```

Comment:
If you change version tag here then you need to change in github releases and tags. So I propose to do not achange it, to provide consistency in history

Reply:
It's already changed in release description